WhaleSpotting/react-app/src/components/Home.tsx

```diff
@@ -1,22 +1,41 @@
+import "../styles/Home.scss";
 import React from "react";
 
-export default function Home(): JSX.Element {
+
+export default function Home() {
```

Comment:
You should say what type this returns
```suggestion
export default function Home(): JSX.Element {
```

Reply:
Fixed

Reply:
fixed